code/game/objects/structures/railings.dm

```diff
@@ -29,13 +29,16 @@
 			to_chat(user, "<span class='warning'>[src] is already in good condition!</span>")
 		return
 
-	if(!(flags_1&NODECONSTRUCT_1))
-		if(I.tool_behaviour == TOOL_WRENCH)
-			to_chat(user, "<span class='notice'>You begin to [anchored ? "unfasten the railing from":"fasten the railing to"] the floor...</span>")
-			if(I.use_tool(src, user, volume = 75, extra_checks = CALLBACK(src, .proc/check_anchored, anchored)))
-				setAnchored(!anchored)
-				to_chat(user, "<span class='notice'>You [anchored ? "fasten the railing to":"unfasten the railing from"] the floor.</span>")
-			return
+///Implements behaviour that makes it possible to unanchor the railing.
+/obj/structure/railing/wrench_act(mob/living/user, obj/item/I)
+	. = ..()
```

Comment:
```suggestion
	. = ..()
	if(.)
		return
```

Reply:
I would argue that the places where this behaviour is implemented are actually doing it wrong.
Adding a component that adds tool behaviour should not block other pre-existing tool behaviours which is exactly what that line would do.